### ISSUE TITLE
Allow multiple Accounts

### ISSUE THREAD

**Issue by ChristianNobis** [john-tornblom/TVHGuide#16]
multiple Accounts would be a great Addition to this app. I'm having a tvheadend running on my raspberry pi just to start a TV-Stream without the need of starting my full featrured Mediacenter and a second running on this mediacenter to watch an record. Fast-switching between theese two would be great

**Comment by rsiebert** [comment]
This is now possible with the new version 1.7.2 on my development branch. 
You can download the latest apk at http://rsiebert.github.io/TVHGuide/

**Comment by ghost** [comment]
I can not install your application. I tried it on android 4.1.2 and 4.3
I have a message application is not installed.

Please compile version apk 1.7.2 with polish language.

**Comment by rsiebert** [comment]
Did you enable the setting to allow installing untrusted applications?
Something like: Settings / Applications / Unknown sources

With that setting on, you can just point your browser on your phone to the url https://github.com/rsiebert/TVHGuide/raw/gh-pages/TVHGuide.apk it should install the apk.

**Comment by ghost** [comment]
Yes i enable the setting to allow installing untrusted applications

I download this app and still i have the same message and app didn't instaled.

**Comment by rsiebert** [comment]
My last apk was not signed. Now I have uploaded a newer version which I have signed myself. Hope it works now.

**Comment by ghost** [comment]
Ok it works :)
After weekend i fix transleted polish :)

**Comment by ghost** [comment]
I installed your version and i have a problem. I can't watch TV. When i click on play i have a ERROR OPENING FILE 66. On oficial app from GOOGLE PLAY i can watch TV with no problem... Please HELP

**Comment by ghost** [comment]
I update polish translated on TVHguide John Tornblom... Update your app.
